index.html

```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
         <li><i class="fa-brands fa-facebook-f"></i></li>
       </ul>
     </section>
+    <section class="card">
+      <div class="grid-items">
+        <div class="cards">
+          <img src="./Images/Snapshoot-Portfolio.png" alt="" />
+        </div>
+        <div class="cards">
+          <h1>Tonic</h1>
+          <div class="cards-details">
+            <div class="project">
+              <span>Canopy</span>
+              <ul class="listA">
+                <li class="listA-items">BackEnd</li>
+                <li class="listA-items">2015</li>
+              </ul>
+            </div>
+
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="listB">
+              <li class="listB-items">HTML</li>
+              <li class="listB-items">Css</li>
+              <li class="listB-items">JavaScript</li>
+            </ul>
+            <button>See Project</button>
```

Comment:
Good job your buttons looks exactly like the template , only make sure to add hover and press effects as requested 

![image (35)](https://github.com/enocol/MobileFirst/assets/59707859/73e0aacc-5512-4a3f-ab10-4c9bdcd1c30d)